README.md

```diff
@@ -1,67 +1,198 @@
 # Rubyric
 
-Rubyric is an online tool in which you can create courses and assignments, 
-receive submissions and assess them by using rubrics. More information about 
+Rubyric is an online tool in which you can create courses and assignments,
+receive submissions and assess them by using rubrics. More information about
 Rubyric and how to use it can be found [here](doc/rubyric.md).
 
-## Install environment
-Install ruby version 2.3.1 or higher, for example with rvm or rbenv. Then
-install rails:
+## Environment
+Install ruby 2.3.1 or higher. One way of installing these versions in Linux is
+by using one of the following Ruby version management tools,
+[rvm or rbenv](https://linuxize.com/post/how-to-install-ruby-on-ubuntu-20-04/).
+
+If you followed the guides provided in the above link. You would find yourself
+adding some scripts to the `~/.bashrc` file. However, you can use your favourite
+shell.
+
+You can always verify your default shell by typing the following command in your
+terminal
+
+```sh
+ls -l /proc/$$/exe
+```
+
+or
+
+```sh
+echo $SHELL
+```
+
+### Ruby and rails environment
+
+Update the RubyGems version
+
+```sh
+sudo gem install rubygems-update
+```
+
+Install rails with the following command
+
 ```sh
 gem install rails -v 5.0.2
 ```
+
 or
+
 ```sh
 sudo gem install rails -v 5.0.2
 ```
-Rubyric uses postgresql as database. Install postgresql and create user.
+
+Before you start *installing any Gem*, it is important to check the Bundler
+version. The current version of Rubyric requires a Bundler version (>= 1.3.0,
+< 2.0). In addition, it is worth noting that Rubyric requires a version
+<= 1.4.X of the [big-decimal](https://github.com/ruby/bigdecimal) gem.
+
+Verify Bundler version
+```sh
+bundler -v
+```
+and/or
+```sh
+gem list bundler
+```
+
+If you have a different Bundler version which is not compatible with the Rubyric
+ecosystem, you must uninstall it and install a compatible version. The command
+for installing the Bundler is similar to the one below. However, you can choose
+the most suitable [version](https://rubygems.org/gems/bundler/versions) for you.
+
+```sh
+gem install bundler -v 1.17.3
+```
+
+### Database
+
+Rubyric uses a PostgreSQL database. Therefore, you must install PostgreSQL and
+create an user.
```

Comment:
"an user" -> "a user" is correct since the pronunciation begins with a consonant